src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -20,6 +20,10 @@
         padding: 0;
     }
 
+    .bcs-AnnotationActivity-menu {
+        top: 0;
```

Comment:
What's the purpose behind this change?

Reply:
When Annotation is rendered within the `ActivityThread` styling is a bit different and this change is to compensate that, so that the menu button is displayed in the same position.

As we haven't tested Annotations in Canvas before, we haven't caught that before.